test/OpenTelemetry.Instrumentation.AWSLambda.Tests/AWSLambdaWrapperTests.cs

```diff
@@ -176,32 +150,6 @@ public async Task TraceAsyncWithInputAndNoReturn(bool setCustomParent)
             this.AssertSpanAttributes(activity);
         }
 
-        [Theory]
-        [InlineData(false)]
-        [InlineData(true)]
-        public async Task TraceAsyncWithNoInputAndNoReturn(bool setCustomParent)
-        {
-            var processor = new Mock<BaseProcessor<Activity>>();
-
-            using (var tracerProvider = Sdk.CreateTracerProviderBuilder()
-                .AddAWSLambdaConfigurations()
-                .AddProcessor(processor.Object)
-                .Build())
-            {
-                var parentContext = setCustomParent ? CreateParentContext() : default;
-                await AWSLambdaWrapper.TraceAsync(tracerProvider, this.sampleHandlers.SampleHandlerAsyncNoInputAndNoReturn, this.sampleLambdaContext, parentContext);
```

Comment:
Please, remove corresponding handler sample SampleHandlerAsyncNoInputAndNoReturn.

Reply:
Done in 0e59d71